app/src/main/java/ch/epfl/sdp/game/data/Participation.kt

```diff
@@ -0,0 +1,11 @@
+package ch.epfl.sdp.game.data
+
+import ch.epfl.sdp.lobby.PlayerParametersFragment
+import ch.epfl.sdp.user.User
+
+class Participation (
+        var user: User = User(),
```

Comment:
Yeah okay. Then class User will need a companion object as a singleton currentUser in US-8